lib/article_json/export/apple_news/elements/list.rb

```diff
@@ -0,0 +1,71 @@
+module ArticleJSON
+  module Export
+    module AppleNews
+      module Elements
+        class List < Base
+          include ArticleJSON::Export::Common::HTML::Elements::Base
+          include ArticleJSON::Export::Common::HTML::Elements::Text
+
+          # List
+          # @return [Hash]
+          def export
+            {
+              role: 'body',
+              text: list_text,
+              format: 'html',
+              layout: 'bodyLayout',
+              textStyle: 'bodyStyle',
+            }
+          end
+
+          private
+
+          # Get the exporter class for text elements
+          #
+          # @return [ArticleJSON::Export::Common::HTML::Elements::<Class>]
+          def text_exporter
+            self.class.exporter_by_type(:text)
+          end
+
+          # When it is an unordered list wrap it in <ul></ul>
+          # When it is an ordered list wrap it in <ol></ol>
+          #
+          # List Text
+          # @return [String]
+          def list_text
+            prepend_list_tag + list + append_list_tag
+          end
+
+          # Each list item should be wrapped in <li></li>
+          #
+          # @return [String]
+          def list
+            @element.content.map do |paragraph_element|
+              line_item = paragraph_element.content.map do |text_element|
+                text_exporter.new(text_element).export
+              end
+              .join
```

Comment:
I would do `end.join` to be consistent with the line 50.

Reply:
Amended